packages/orbit-components/src/Dialog/index.jsx

```diff
@@ -174,7 +174,11 @@ const Dialog = ({
         <StyledDialogContent shown={shown} ref={ref} id={dialogID}>
           {illustration && <IllustrationContainer>{illustration}</IllustrationContainer>}
           <Stack spacing="XSmall" spaceAfter="medium">
-            {title && <Heading type="title3">{title}</Heading>}
+            {title && (
+              <Heading type="title3" align="center" largeMobile={{ align: "start" }}>
+                {title}
+              </Heading>
+            )}
```

Comment:
I think it's not related to implementation in component. I guess Will and Milan meant to change it inside storybook and documentation, because it's a `React.Node` prop and you can pass there `ButtonLink` or `Button`